### DISCUSSION TITLE
Locale Inconsistency when using labelProps

### DISCUSSION THREAD

**Discussion by wilsonadenuga** [gpbl/react-day-picker#2541] [Translation and localization]
When i use labelProps as defined in the docs, https://daypicker.dev/docs/translation#example-italian-labels

the aria-label for my days are now in English (excluding selected and today) , I had to fix like so 

```tsx
labels={{
				labelDayButton: (date, { today, selected }) => {
					let label = format(date, 'PPPP', { locale: it });
					if (today) label = `Oggi, ${label}`;
					if (selected) label = `${label}, selezionato`;
					return label;
				},
				labelNext: () => 'Vai al mese successivo',
				labelPrevious: () => 'Vai al mese precedente',
			}}`
```
	
notice the local I passed in let label = ......, this is missing from the docs.

PS: I can update the docs with this information if you don't mind

**Comment by gpbl** [comment]
Do you mean `format()` call is missing `{ locale: it }` - right?

**Comment by wilsonadenuga** [comment]
Yeah, correct
